omnibus/config/projects/chef.rb

```diff
@@ -20,7 +20,7 @@
 homepage "https://www.chef.io"
 
 build_iteration 1
-build_version "12.6.0"
+build_version "12.7.0"
```

Comment:
or the top level `VERSION` file, should be just as authoritative and you can just do `IO.read(file).strip` and be done with it.